gems/sorbet-runtime/lib/types/configuration.rb

```diff
@@ -363,6 +363,22 @@ def self.scalar_types
     @scalar_types || @default_scalar_types
   end
 
+  @module_name_mangler = nil
```

Comment:
Why not make the default something like:
```suggestion
  @module_name_mangler = ->(type) { MODULE_NAME.bind(type).call }
```
so that you don't need the `if mangler` check in `serde_transform.rb` and can unconditionally do `T::Configuration.module_name_mangler.call(type)`?

Reply:
I considered that, but wanted to preserve the ability to go back to the default implementation with `module_name_mangler = nil`

Reply:
You can still do something like what `scalar_types` does:
```ruby
@default_module_name_mangler = ->(type) { MODULE_NAME.bind(type).call }
@module_name_mangler = nil

def self.module_name_mangler
  @module_name_mangler || @default_module_name_mangler
end

def self.module_name_mangler=(handler)
  @module_name_mangler = handler
end
```
which will allow you to get back to the default implementation with `module_name_mangler = nil`.